.vscode/settings.json

```diff
@@ -82,5 +76,6 @@
       "language": "markdown",
       "scheme": "file"
     }
-  ]
+  ],
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
This is useful to ensure that everyone is using the locally installed typescript version. Kind of like Corepack + `package.json#packageManager`